### DISCUSSION TITLE
Server crashes with 1.2 Beta 3 and Acustica

### DISCUSSION THREAD

**Discussion by bluesbop** [apohl79/audiogridder#563] [Help]
I tried using Beta 3 with a Logic project I had created using AG 1.1.1 but the server just kept crashing and reopening over and over. If I delete the instance of the Gridder AU plug on the Submix bus (which had 1 IK Tape and 2 Acustica plugs) and open a blank one, I can't add any Acustica plugins, just ones from other manufacturers. If I open a Gridder plug in an individual channel or a different bus, OTOH, I am able to open Acustica plugins. 

Log and trace files are attached, will send the core dump and crash report via Wetransfer.

Thanks, 
Michael

[FX.zip](https://github.com/apohl79/audiogridder/files/6497654/FX.zip)
[Server.zip](https://github.com/apohl79/audiogridder/files/6497655/Server.zip)

**Comment by CaBuffalo** [comment]
Hi, Michael,

IK tape is a tricky one. It's not very stable with AG at this moment.
Acustica has known compatibility issues running with AG.

If it works on its own instance, keep that way. That's what I've been doing
currently.

On Mon, May 17, 2021 at 6:07 PM bluesbop ***@***.***> wrote:

> I tried using Beta 3 with a Logic project I had created using AG 1.1.1 but
> the server just kept crashing and reopening over and over. If I delete the
> instance of the Gridder AU plug on the Submix bus (which had 1 IK Tape and
> 2 Acustica plugs) and open a blank one, I can't add any Acustica plugins,
> just ones from other manufacturers. If I open a Gridder plug in an
> individual channel or a different bus, OTOH, I am able to open Acustica
> plugins.
>
> Log and trace files are attached, will send the core dump and crash report
> via Wetransfer.
>
> Thanks,
> Michael
>
> FX.zip <https://github.com/apohl79/audiogridder/files/6497654/FX.zip>
> Server.zip
> <https://github.com/apohl79/audiogridder/files/6497655/Server.zip>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/563>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6A5N7T3ECOTZJGOOY3TOG4VRANCNFSM45BPZ2CA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by bluesbop** [comment]
I've been experimenting more with AG. I had a project (a different one than the one I posted earlier about) with just 3 plugs running inside AG 1.1.1. All was working well until all of a sudden during playback with no plugin windows open the AG server got into the crash/restart loop that I had run into with the other project using 1.2beta3. The only solution was to remove AG, open a clean instance, and reinsert the 3 plugins inside. Once I did that, everything ran fine. I also tried opening up an earlier version of the project, since I do incremental Save As for safeguarding, and the earlier version (v.3) ran fine without having to replace AG. I re-opened the version (v.6) that kept crashing, and it still would not work. V.7, with the replaced AG, still worked fine. I tried the same tactic with another project where the AG server was crashing a lot, and a clean insertion of AG fixed the issue there as well. 

This all leads me to believe that somehow something is getting corrupted in AG AU instances over time and eventually become too unstable to use. I've since reinstalled beta3, and it's working fine as well. The added preset functionality is a huge help, as it's quicker to recall plug chains if I have to open a fresh instance of AG. 

Hope this helps point the developers in the right direction to fix this issue.
